.github/workflows/ci-cd.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Install go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.15.x
+          go-version: 1.16.5
```

Comment:
"1.16.x" instead should always pick latest go 1.16.x?

Reply:
will try with 1.16.x